codegen/src/json_schema.rs

```diff
@@ -42,6 +43,7 @@ pub struct JsonObjectFieldInfo {
 #[derive(Clone, Debug)]
 pub struct JsonEnum {
     pub name: String,
+    pub untagged: bool,
```

Comment:
ah my bad :) i did not realise was an actual schema. duh.

json-schema.org seems quite dense. Differing types works in this case, so it's fine with me if it's fine with you :D